assembl/static2/storybook.test.js

```diff
@@ -0,0 +1,7 @@
+// @flow
+/* eslint-disable */
```

Comment:
same here, disable the error explicitely please

Reply:
fixed